### ISSUE TITLE
[Bug Report][3.5.6] v-text-field: clickable append-inner-icon, we should be able to add aria-label and aria-pressed to inner icons

### ISSUE THREAD

**Issue by maja-fredstrom** [vuetifyjs/vuetify#19278]
### Environment
**Vuetify Version:** 3.5.6
**Vue Version:** 3.4.20
**Browsers:** Chrome 122.0.0.0
**OS:** Windows 10

### Steps to reproduce
Create a simple v-text-field 
append an inner icon to it
add event listener @click:append-inner
listen to the inner icon with a screen reader or check the source code

### Expected Behavior
Clickable inner icon should be accessible by keyboard. 
Screen reader should read a accessible information when inner icon selected. Developer should be able to define aria- attributes.

### Actual Behavior
You can't access to the inner icon navigating with tab button. 

Screen reader reads the label and add "edit action" to it. For example reads "password edit action" instead of "show password, button active/inactive"

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpdUk1zwiAQ/SsrF+pMMdqv6Thq662nXntQDzTZVKYEGCCxjvrfu8Sqsbfl7dv3Fh6LHQs+z+bODZoa2ZhNIlZOy4izpQGYNEI615btIbcmSmXQ/0EtGPEnilKhLk4gQCMqW6CeLlkVvpbs0hiTHppCKEMqQpEgccLabkbwArwqlMAtchifa2HLkl8p+FpjoKnFTdOfzqDXa2C/B+5kCBvrC4jWAin6yGF1NRi3DrtuafHW6jR65WNkldjKuDqK0VCMuk0tP9vbvVtfSQ0bFdewViZC0uwSE0i8eQSNMkR4hnwtvcwj+tDl5bY2hF2A11yr/Pvquc67T6HXFheB7JRR9j+khBwznGSdbOkYcq9chICxPkasKkevBjvwWMIBSm8r4PQteKKnFQ1dgPIkf2Lc8DfU2sKH9bro8f6FcloykUqpA1Jvkh3tyIgdbtn94GFwN2SpeBw8sdUv9oXHXA==)

### Other comments
I have tried to add the inner-icon as append inner slot. The problem is that the append-inner slot doesn't inherit the input color.
